docs/list_objects/pipeline/pipeline_algorithm.md

```
@@ -0,0 +1,175 @@
+    define eight: ten or three from parent
+    define parent: [org]
+
+type object
+    relations
+    define zero: [org#one, team#eight]
+```
+
+This model can also be represented as a graph:
+
+<p align="center">
+    <img src="list_objects_pipeline_model_with_types.png" height="1000">
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)